tutorials/Coordinates/Coordinates.ipynb

```diff
@@ -0,0 +1,781 @@
+     ],
+     "language": "python",
+     "metadata": {},
+     "outputs": []
+    },
+    {
+     "cell_type": "markdown",
+     "metadata": {},
+     "source": [
+      "If the object you're interested in is in SIMBAD, you can also look it up directly from its name using the `SkyCoord.from_name()` [class method](https://julien.danjou.info/blog/2013/guide-python-static-class-abstract-methods). \n",
+      "\n",
+      "(Note that this requires an internet connection.  It's safe to skip if you don't have one, because we defined it above explicitly.)"
+     ]
+    },
+    {
+     "cell_type": "code",
+     "collapsed": false,
+     "input": [
+      "hcg7_center = SkyCoord.from_name('HCG 7')\n",
+      "hcg7_center"
+     ],
+     "language": "python",
+     "metadata": {},
+     "outputs": []
+    },
+    {
+     "cell_type": "markdown",
+     "metadata": {},
+     "source": [
+      "This object we just created has various useful ways of accessing the information contained within it.  In particular, the ``ra`` and ``dec`` attributes are specialized ``Quantity`` objects that store angles and provide pretty representations of those angles and some useful attributes to quickly convert to common angle units:"
+     ]
+    },
+    {
+     "cell_type": "code",
+     "collapsed": false,
+     "input": [
+      "type(hcg7_center.ra), type(hcg7_center.dec)"
+     ],
+     "language": "python",
+     "metadata": {},
+     "outputs": []
+    },
+    {
+     "cell_type": "code",
+     "collapsed": false,
+     "input": [
+      "hcg7_center.dec"
+     ],
+     "language": "python",
+     "metadata": {},
+     "outputs": []
+    },
+    {
+     "cell_type": "code",
+     "collapsed": false,
+     "input": [
+      "hcg7_center.ra"
+     ],
+     "language": "python",
+     "metadata": {},
+     "outputs": []
+    },
+    {
+     "cell_type": "code",
+     "collapsed": false,
+     "input": [
+      "hcg7_center.ra.hour"
+     ],
+     "language": "python",
+     "metadata": {},
+     "outputs": []
+    },
+    {
+     "cell_type": "markdown",
+     "metadata": {},
+     "source": [
+      "Now that we have a `SkyCoord` object, we can start using it to access data from the [Sloan Digitial Sky Survey](http://www.sdss.org/).  Lets start by just trying to get a picture using the SDSS image cutout service."
+     ]
+    },
+    {
+     "cell_type": "code",
+     "collapsed": false,
+     "input": [
+      "impix = 1024\n",
+      "imsize = 12*u.arcmin\n",
+      "cutoutbaseurl = 'http://skyservice.pha.jhu.edu/DR12/ImgCutout/getjpeg.aspx'\n",
+      "cutouturl = cutoutbaseurl + '?' + urllib.urlencode(dict(ra=hcg7_center.ra.deg, dec=hcg7_center.dec.deg, \n",
+      "                                                        width=impix, height=impix, \n",
+      "                                                        scale=imsize.to(u.arcsec).value/impix))\n",
+      "urllib.urlretrieve(cutouturl, 'HCG7_SDSS_cutout.jpg')"
```

Comment:
I think it should, otherwise why are we shipping something like `download_file` in the first place?

urllib is confusing.  requests is less confusing but not default installed (and even then confusing to a newbie--I like the brain dead simplicity of download_file).

download_file returns the absolute path to the file that was downloaded.  Just do somethng like `path_to_image = download_file(...)` and use `path_to_image`.